### ISSUE TITLE
Asc/Desc control labels showing

### ISSUE THREAD

**Issue by crstauf** [johnbillion/query-monitor#221]
When viewing on the frontend, if a theme does not have `.screen-reader-text` in their CSS, the labels for the ascending and descending controls show:

![screen shot 2016-08-03 at 6 04 32 pm](https://cloud.githubusercontent.com/assets/4573033/17383621/e23580ae-59a4-11e6-9c95-0470e28ada59.png)

recommend adding the class to the stylesheet specifically within QM's output.

**Comment by johnbillion** [comment]
Good spot. This happens on the login screen too.

**Comment by johnbillion** [comment]
Fixed in #232
